llvmlite/tests/test_binding.py

```diff
@@ -3020,20 +3040,29 @@ class TestNewModulePassManager(BaseTest, NewPassManagerMixin):
     def pm(self):
         return llvm.create_new_module_pass_manager()
 
-    def test_close(self):
-        mpm = self.pm()
-        mpm.close()
-
-    def test_run(self):
-        pb = self.pb(speed_level=3, size_level=0)
+    def run_o_n(self, level):
         mod = self.module()
-        orig_asm = str(mod)
+        orig_asm = str(self.module())
```

Comment:
Does this not work if it's `str(mod)` for some reason I've missed?

Reply:
Fixed it.